test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -204,9 +206,9 @@ def test_pageserver_respects_overridden_resident_size(eviction_env: EvictionEnv)
         du_by_timeline[large_tenant] - du_by_timeline[small_tenant] > 5 * env.layer_size
     ), "ensure this test will do more than 1 eviction"
 
-    # give the larger tenant a haircut while prevening the smaller tenant from getting one
+    # give the larger tenant a haircut while preventing the smaller tenant from getting one
     min_resident_size = du_by_timeline[small_tenant]
-    target = du_by_timeline[large_tenant] - du_by_timeline[small_tenant]
+    target = (du_by_timeline[large_tenant] - du_by_timeline[small_tenant]) - 2*env.layer_size
```

Comment:
This also has a style error from `black`, but what's the point here with the `- 10MiB`?

Reply:
699ca672a48c307355960924f8b8e767e68ea508 looks very good!